core/database/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+require("@arangodb/aql/cache").properties({ mode: "demand" });
+
+const   createRouter = require('@arangodb/foxx/router');
+const   router = createRouter();
+
+router.use( "/acl", require('./api/acl_router') );
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ORNL/DataFed2/pullRequest?prid=8791729)
```suggestion
router.use( "/acl", require("./api/acl_router") );
```